test/linalg/bidiag.jl

```diff
@@ -97,29 +97,30 @@ srand(1)
         end
 
         @testset "triu and tril" begin
+            bidiagcopy(dv, ev, uplo) = Bidiagonal(copy(dv), copy(ev), uplo)
```

Comment:
were Tridiagonal or SymTridiagonal behaving like that too, or was Bidiagonal the only case of it?